fala/apps/adviser/tests/test_cookies_page_journeys_playwright.py

```diff
@@ -0,0 +1,86 @@
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from playwright.sync_api import sync_playwright, expect
+from fala.playwright.setup import PlaywrightTestSetup
+
+
+class CookiesPageEndToEndJourneys(PlaywrightTestSetup):
+    def test_cookies_page_view_accesed_from_footer(self):
+        page = self.visit_cookies_page_from_footer()
+        expect(page.h1).to_have_text("Cookies")
+
+    def test_cookies_page_view_accesed_from_banner(self):
+        page = self.visit_cookies_page_from_banner()
+        expect(page.h1).to_have_text("Cookies")
+
+
+class CookiesPageEndToEndJourneysWithFreshSetUp(StaticLiveServerTestCase):
+    def test_that_cookie_has_appropriate_value(self):
```

Comment:
For now this is fine for this part of the cookie work. I'll add this to the next cookie banner work story. 

This is a really big test which can be broken up in re-useable functions. So what I'm thinking, this could be 
broken down into three tests that can be used as functions in a much bigger test.

- Accept cookie banner, cookie is made.
- Reject cookie banner, cookie is not added to session.
- Accept Cookie on cookie policy page. Cookie is made.
- Reject cookie on cookie policy page. Cookie is rejected.
- Now using the above tests as functions you can combine them all into a nice flow test. I accepted cookie in banner, go to cookie page, I reject cookie. You can then even have a test that flips the logic, reject then accept, etc. 

This provides better test coverage as you've have individual part of the components and it's journey.